### PR TITLE
implemented form to add movie

### PR DESCRIPTION
**Pull request by tiserett**
[DEMO LINK](https://tiserett.github.io/react_movies-list-add-form/)

### CODE REVIEW

File: src/components/NewMovie/NewMovie.tsx [hunk 1]
```diff
@@ -1,45 +1,99 @@
...
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+
+  const valid = (!title || !imgUrl || !imdbId || !imdbUrl);
```
Comment (Githi54) [inline src/components/NewMovie/NewMovie.tsx:19]:
```suggestion
  const valid = !title || !imgUrl || !imdbId || !imdbUrl;
```

File: src/components/NewMovie/NewMovie.tsx [hunk 2]
```diff
@@ -1,45 +1,99 @@
...
+  const setDefault = () => {
+    setTitle('');
+    setDescription('');
+    setImgUrl('');
+    setImdbUrl('');
+    setImdbId('');
+  };
+
+  const handleEmptyInput = (value: string, callback: (val: string) => void) => {
+    if (!value.trim()) {
```
Comment (Githi54) [inline src/components/NewMovie/NewMovie.tsx:30]:
```suggestion

```

**Review by Githi54 — APPROVED** [review]
Well done!🚀

File: src/components/NewMovie/NewMovie.tsx [hunk 3]
```diff
@@ -1,45 +1,99 @@
...
+    });
+
+    setDefault();
+  };
 
   return (
-    <form className="NewMovie" key={count}>
+    <form
+      className="NewMovie"
+      key={count}
```
Comment (AlinaMatuschak) [inline src/components/NewMovie/NewMovie.tsx:55]: Why did you add key for form?

File: src/components/NewMovie/NewMovie.tsx (hunk 2 above)
Comment (AlinaMatuschak) [reply src/components/NewMovie/NewMovie.tsx:30]: That’s why I can’t empty input

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
by some reason, I can’t delete the last element on the input
Screenshot 2022-09-13 at 21 07 57

File: src/components/NewMovie/NewMovie.tsx (hunk 3 above)
Comment (tiserett) [reply src/components/NewMovie/NewMovie.tsx:55]: it was there by default, thought it was there for tests
